git_machete/tests/functional/test_machete.py

```diff
@@ -4,193 +4,74 @@
 import re
 import string
 import sys
+import textwrap
 import time
 import unittest
 
 from git_machete import cmd
 
 
 class SandboxSetup:
-
     def __init__(self) -> None:
         self.file_dir = os.path.dirname(os.path.abspath(__file__))
-        self.remote_path = os.popen('mktemp -d').read().strip()
-        self.sandbox_path = os.popen('mktemp -d').read().strip()
+        self.remote_path = os.popen("mktemp -d").read().strip()
```

Comment:
Actually... any reason for preferring `"` over `'`? 🤔 or just for the sake of consistency?

Reply:
Oh shoot, this is a result of running [`black`](https://github.com/psf/black) over the file while I was editing.
I don't have a strong preference here, though a zero-configuration `black` run is always nice.

Reply:
Okay, I didn't know that tool myself... do you think it'd make sense to replace pep8 check with a `black --check` in tox config?

Reply:
I'd be in favor of that, I typically configure `black` and `isort` via [`pre-commit`](https://pre-commit.com/) hooks, then verify the pre-commit runs in `tox`. I'd be happy to open a follow-up PR for that here if you'd like.

Reply:
Roger. Would you prefer a follow-up or a merge into this PR?

Reply:
In the spirit of git-machete... I'd prefer a separate PR ;)